quickstart.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# try to install everything needed to run
+
+if [ -f "/etc/arch-release" ]; then
+  # TODO packages for arch linux
+  echo "::: Arch linux detected but installing the basic dependencies is not yet implemented"
+elif [ -f "/etc/debian_version" ]; then
+  echo "::: Debian based linux detected, updating system"
+  sudo apt-get update
+  echo "::: Installing basic programs"
+  sudo apt install -y git curl python3.7 python3-pip
```

Comment:
You cant install Python3.7 via apt with the default source list of debian.

Reply:
Oh, I see. On Ubuntu it works though. My solution to this problem is that if you are on Ubuntu it gets installed the way it is and if not the user will just be warned and it will be assumed that Python 3.7 ist installed. I think compiling it from source would just be too much. 
But if you have a better idea please let me know.